{{ cookiecutter.project_slug }}/.github/workflows/ci.yml

```
@@ -9,7 +9,7 @@ on:
 
 jobs:
   test:
-    runs-on: ${{ "{{" }}matrix.os{{ "}}" }}
+    runs-on: ${{ matrix.os }}
```

We need these because cookiecutter will not correctly interpret this because the jinja syntax is used by cookiecutter


```suggestion
    ${{ "{{" }}matrix.os{{ "}}" }}
```